src/type/type.js

```diff
@@ -84,8 +84,10 @@ function stringToGraphQLType(type) {
  * @return {Object}
  */
 function listToGraphQLEnumType(list, name) {
-  const values = {};
-  list.forEach((val) => { values[val] = {value: val}; });
+  const values = reduce(list, (values, val) => {
```

Comment:
It works on my project.
I do it like in `getOrderByType` method, which uses `lodash` reduce. 
May rewrite on regular ES6 reduce.

PS. a subtle hint for tests :P